whylog/rulesbase/__init__.py

```diff
@@ -1,6 +1,60 @@
+import yaml
+from abc import ABCMeta, abstractmethod
 
+class AbstractConfigAccess(object):
 
-class WhylogBase(object):
+        """docstring for AbstractConfigAccess"""
```

Comment:
use 4 spaces like the rest of the project, please. Also, unless a docstring is necessary, I think we can skip it.